iron-menu-behavior.html

```diff
@@ -161,11 +168,19 @@
     _focusNext: function() {
       var length = this.items.length;
       var curFocusIndex = Number(this.indexOf(this.focusedItem));
+      var owner = Polymer.dom(this).getOwnerRoot() || document;
```

Comment:
I think the owner needs to be computed for each item rather than the menu itself: if the items are distributed from above, their owner won't be the menu. ([example](http://jsbin.com/foceho/1/edit?html,output))